src/AbstractPolyhedron_functions.jl

```diff
@@ -318,6 +318,22 @@ function linear_map(M::AbstractMatrix{N},
     end
 end
 
+# handle different numeric types
+function linear_map(M::AbstractMatrix{NM},
+                    P::AbstractPolyhedron{NP};
+                    check_invertibility::Bool=true,
+                    cond_tol::Number=DEFAULT_COND_TOL,
+                    use_inv::Bool=!issparse(M)
```

Comment:
Can we use `kwargs...` instead? I prefer spelling out these arguments and their default values in one place only. This makes changing things later much easier.

Reply:
Agreed. I changed to use `kwargs...`